Assets/AppCenter/Plugins/AppCenterSDK/Crashes/Shared/Crashes.cs

```diff
@@ -78,11 +78,20 @@ public static void OnHandleUnresolvedException(object sender, UnhandledException
             if (exception != null)
             {
                 Debug.Log("Unhandled exception: " + exception.ToString());
+#if UNITY_IOS && !UNITY_EDITOR
+                var exceptionWrapper = CreateWrapperException(exception);
+                var errorId = CrashesInternal.TrackException(exceptionWrapper.GetRawObject(), null, null);
+                if (_enableErrorAttachmentsCallbacks)
+                {
+                    SendErrorAttachments(errorId);
```

Comment:
Wait. Why sending attachments is called at the crash time?

Reply:
When the main thread don't crash then attachments should be sent